holoviews/plotting/bokeh/links.py

```diff
@@ -182,6 +182,24 @@ def __init__(self, root_model, link, source_plot, target_plot):
                     ax.reset_end = end
 
         tool = RangeTool(**axes)
+
+        use_handles = getattr(link, 'use_handles', True)
+        if use_handles and bokeh35:
+            tool.overlay.use_handles = use_handles
+            tool.overlay.handles.all.hover_fill_color = "grey"
+            tool.overlay.handles.all.hover_fill_alpha = 0.25
+            tool.overlay.handles.all.hover_line_alpha = 0
+            tool.overlay.handles.all.fill_alpha = 0.1
+            tool.overlay.handles.all.line_alpha = 0.25
+
+        start_gesture = getattr(link, 'start_gesture', 'tap')
+        if start_gesture is not None and bokeh35:
```

Comment:
Would this ever be None if the getattr default is tap?

Reply:
thanks for the feedback. I think you're right, I'm not sure from which condition I was previously thinking that None might be the value. I committed a change